packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.html

```diff
@@ -0,0 +1,138 @@
+                            data-name="{{:: preset.id }}"
+                            data-model="$ctrl.model.imageId"
+                            data-label="{{:: preset.description }}"
+                            data-variant="light"
+                            data-values="[preset]"
+                            data-required>
+                        </oui-select-picker>
+                    </div>
+                </div>
+            </div>
+
+            <div data-ng-if="$ctrl.model.workflowTemplate == $ctrl.BUILD_IMAGE">
+                <span data-translate="pci_projects_project_serving_namespace_models_add_storage_path_info"></span>
+
+                <oui-field data-size="xl">
+                    <oui-select
+                        disabled="$ctrl.paths.length == 0"
+                        id="storagePath"
+                        name="storagePath"
+                        model="$ctrl.model.storagePath"
+                        items="$ctrl.paths"
+                        data-required>
+                    </oui-select>
+                </oui-field>
+                <p data-ng-if="!$ctrl.paths.length">
+                    <span class="text-danger" ng-click="$ctrl.goToContainer()"
+                          data-translate="pci_projects_project_serving_namespace_models_add_storage_path_no_path"></span>
+                </p>
+            </div>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_name' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+
+            <oui-field  data-size="xl"
+                        data-help-text="{{ :: 'pci_projects_project_serving_namespace_models_add_name_help' | translate }}">
+                <input class="oui-input" type="text"
+                       id="name" name="id"
+                       data-ng-model="$ctrl.model.id"
+                       data-required
+                       data-ng-pattern="/^[\w-]+$/"/>
+            </oui-field>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_autoscaling' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+
+            <span data-translate="pci_projects_project_serving_namespace_models_add_autoscaling_info"></span>
+
+            <oui-field data-label="{{:: 'pci_projects_project_serving_namespace_models_storage_scale_min' | translate }}"
+                       data-size="auto">
+                <oui-numeric min="1" max="1" model="$ctrl.core" name="test" disabled>
+                </oui-numeric>
+            </oui-field>
+            <oui-field data-label="{{:: 'pci_projects_project_serving_namespace_models_storage_scale_max' | translate }}"
+                       data-size="auto">
+                <oui-numeric min="3" max="3" model="$ctrl.max" name="test2" disabled>
```

Comment:
Please use `data-` for properties that don't belong to `HTML5`, for example `model`